saleor/account/events.py

```diff
@@ -14,6 +14,32 @@ def customer_account_created_event(*, user: User) -> Optional[CustomerEvent]:
     return CustomerEvent.objects.create(user=user, type=CustomerEvents.ACCOUNT_CREATED)
 
 
+def customer_account_activated_event(
+    *, staff_user: UserType, app: AppType, account_id: int
+) -> Optional[CustomerEvent]:
+    if not user_is_valid(staff_user):
+        staff_user = None
+    return CustomerEvent.objects.create(
+        user=staff_user,
+        app=app,
+        type=CustomerEvents.ACCOUNT_ACTIVATED,
+        parameters={"account_id": account_id},
+    )
+
+
+def customer_account_deactivated_event(
+    *, staff_user: UserType, app: AppType, account_id: int
+) -> Optional[CustomerEvent]:
+    if not user_is_valid(staff_user):
+        staff_user = None
+    return CustomerEvent.objects.create(
+        user=staff_user,
+        app=app,
+        type=CustomerEvents.ACCOUNT_DEACTIVATED,
+        parameters={"account_id": account_id},
```

Comment:
Don't you also need to extend `CustomerEvent` type to provide field `accountID`?

Reply:
If yes, just reminder that the `id` should be converted to graphql form

Reply:
Should it? I think for other types we store database ID. We should rather translate it in some resolver in the corresponding GraphQL type.